nixos/modules/services/hardware/usbmuxd.nix

```diff
@@ -0,0 +1,74 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  defaultUserGroup = "usbmux";
+  apple = "05ac";
+
+  cfg = config.services.usbmuxd;
+
+in
+
+{
+  options.services.usbmuxd = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable the usbmuxd ("USB multiplexing daemon") service. This daemon is
+        in charge of multiplexing connections over USB to an iOS device. This is
+        needed for transferring data from and to iOS devices (see ifuse). Also
+        this may enable plug-n-play tethering for iPhones.
+      '';
+    };
+
+    user = mkOption {
```

Comment:
Is there a need to have user+group configurable?

Reply:
usbmuxd does not create any files in the filesystem (other than a socket and a pid file at runtime). The reason I added a user option is because usbmuxd has a specific argument for that (-U). I don't think there's any point in making the group configurable.

Reply:
Can you try it out with ```DynamicUser = true;``` then?

Reply:
This won't work for the same reason as mentioned here: https://github.com/NixOS/nixpkgs/pull/30518#discussion_r145327722